.github/workflows/windows.yml

```diff
@@ -36,6 +37,9 @@ jobs:
       - name: Add msbuild to PATH
         uses: microsoft/setup-msbuild@v2
 
+      - name: Install NASM to support SIMD optimizations 
+        uses: ilammy/setup-nasm@v1
+
```

Comment:
I'd like to call attention to this change, because I made it in response to a warning message that I saw while debugging. It said that SIMD wasn't being enabled because NASM wasn't found, and that "performance will suffer".

Reply:
It might make sense to move that to a different PR.